opm/output/eclipse/EclipseIO.cpp

```diff
@@ -416,9 +409,11 @@ void EclipseIO::writeTimeStep(int report_step,
                               double secs_elapsed,
                               data::Solution cells,
                               data::Wells wells,
-                              std::map<std::string, double> misc_summary_values,
-                              std::map<std::string, std::vector<double>> extra_restart,
-			      bool write_double)
+                              const std::map<std::__cxx11::string, double>& single_summary_values,
```

Comment:
We can't use symbols like `std::__cxx11::string` in our interfaces.  Those are in the implementation's (i.e., compiler + library's) namespace.  Is there a reason `std::string` cannot be used?

Reply:
I didn't notice. I must have been changed automatically by qt-creator. It has a convenient (but it seems not very bullet prof) option for synchronizing the .cpp and .hpp file. I will revert this back to std::string.